src/disk_interface_test.cc

```diff
@@ -188,6 +188,19 @@ TEST_F(DiskInterfaceTest, ReadFile) {
   EXPECT_EQ("", err);
 }
 
+TEST_F(DiskInterfaceTest, WriteFile) {
```

Comment:
This test is not passed on Windows (but passed on Linux) without the fix.
Long file name + some characters for the path overflow MAX_PATH